imod/mf6/model.py

```diff
@@ -190,14 +191,14 @@ def __get_domain_geometry(
         idomain = discretization["idomain"]
         return top, bottom, idomain
 
-    def __get_k(self):
-        try:
-            npf = self[imod.mf6.NodePropertyFlow._pkg_id]
-        except RuntimeError:
-            raise ValidationError("expected one package of type ModePropertyFlow")
-
-        k = npf["k"]
-        return k
+    def _get_k(self):
+        npf_key = self._get_pkgkey(imod.mf6.NodePropertyFlow._pkg_id)
+        if not npf_key:
+            raise KeyError(
+                "No NodePropertyFlow package was assigned to Modflow6Model, make sure this is included."
```

Comment:
Would the following error message be better?
`Unable to obtain the hydraulic conductivity. Make sure a NodePropertyFlow package isassigned to the Modflow6Model`

Reply:
Yes that is better! Implemented that message.